### PR TITLE
Add "*.txt" files in path for GH workflow.

### PR DESCRIPTION
**Pull request by RaghavPrabhakar66**
## Description
Added "*.txt" files as a path in tests.yml to trigger the GH workflow.

Fixes issue #72.

## Affected Dependencies
None

## How has this been tested?
None

## Checklist
- [x ] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [ ] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [ ] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [ ] My changes are covered by tests

### CODE REVIEW

**Comment by arturomf94** [comment]
Is this to trigger the workflow whenever there are changes in the requirments?

**Review by arturomf94 — CHANGES_REQUESTED** [review]
I think this is great but maybe it's better to make the file more explicit: 

i.e.

Have `requirements_dev.txt` / `requirements.txt` instead of `**.txt`.

Because I don't think we want to trigger the workflow if we change anything in the license, for instance.

**Comment by arturomf94** [comment]
> I think this is great but maybe it's better to make the file more explicit:
> 
> i.e.
> 
> Have `requirements_dev.txt` / `requirements.txt` instead of `**.txt`.
> 
> Because I don't think we want to trigger the workflow if we change anything in the license, for instance.

@gmuraru what do you think about this?

**Comment by RaghavPrabhakar66** [comment]
@gmuraru  arturomf94 I have another doubt, Issue #76 states that we remove all dependencies from `requirements.txt` and put them to `setup.cfg` and to remove `requirements.txt` from GA workflows. Isn't that issue in contradiction to this one?

**Comment by arturomf94** [comment]
> @gmuraru @arturomf94 I have another doubt, Issue #76 states that we remove all dependencies from `requirements.txt` and put them to `setup.cfg` and to remove `requirements.txt` from GA workflows. Isn't that issue in contradiction to this one?

That's a good point! 🤔 @gmuraru?

**Comment by gmuraru** [comment]
There is also another requirement file named ```requirements_dev.txt```.
Ideally, we would leave only the ```setup.cfg``` needed for installing sympc and the ```requirements_dev.txt``` would be used to install the devs stuff.

**Comment by arturomf94** [comment]
@RaghavPrabhakar66 - I think, for now, let's add both files explicitly: requirements_dev.txt and requirements.txt :+1:

**Comment by RaghavPrabhakar66** [comment]
@gmuraru @arturomf94 I have both requirements.txt to GA path.

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -8,13 +8,16 @@ on:
...
+      - "requirements_dev.txt"
+      - "requirements.txt"
 
   pull_request:
     types: [opened, synchronize, reopened]
     paths:
       - "**.py"
       - "setup.cfg"
       - ".github/workflows/**.yml"
+      - "**.txt"
```
Comment (gmuraru) [inline .github/workflows/tests.yml:20]: Q: Could you also change it here?

Comment (RaghavPrabhakar66) [reply .github/workflows/tests.yml]: Done.

**Comment by arturomf94** [comment]
Thank you @RaghavPrabhakar66. This looks great IMO. Unfortunately, the tests are currently failing but I believe it has nothing to do with your changes. I think it's best if we try to wait until these problems are fixed anyway and we could merge after that.

**Comment by arturomf94** [comment]
Hm... That's really weird. I don't know what could be going on here. @gmuraru - any ideas?

**Comment by gmuraru** [comment]
> Hm... That's really weird. I don't know what could be going on here. @gmuraru - any ideas?

I re-run the tests now to see if it something that happens only sometimes (we should open a ticket if that is the case)

**Review by gmuraru — APPROVED** [review]
LGTM! We need to keep an eye on that tolerance tests

**Review by arturomf94 — APPROVED** [review]
:)

**Pull request merged**